calculator/pom.xml

```diff
@@ -149,6 +149,13 @@
             <groupId>com.market.banica</groupId>
             <artifactId>common</artifactId>
         </dependency>
+
+        <dependency>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-surefire-plugin</artifactId>
+            <version>3.0.0-M4</version>
```

Comment:
please extract the version in properties

Reply:
Fixed